pkg/handler/api.go

```diff
@@ -74,9 +72,7 @@ func RegisterRoutes(engine *echo.Echo) {
 		RegisterRepositoryRoutes(group, &daoRepo, &introspectRequest)
 		RegisterRepositoryParameterRoutes(group, &daoRepo, &externalRepo)
 
-		daoRpm := dao.GetRpmDao(db.DB, &dao.RpmDaoOptions{
-			PagedRpmInsertsLimit: pointy.Int(pagedRpmInsertsLimit),
-		})
+		daoRpm := dao.GetRpmDao(db.DB, &dao.RpmDaoOptions{})
```

Comment:
As far as I can tell, RpmDaoOptions is no longer being used anywhere, so I think that can be removed from here and wherever else it still remains,